core/src/main/scala/knobs/Resource.scala

```diff
@@ -146,11 +150,9 @@ object Resource {
   val notificationPool = pool("knobs-notification-pool")
 
   private val watchService: WatchService = FileSystems.getDefault.newWatchService
-  private val watchProcess: Process[Task, WatchEvent[_]] =
-    Process.eval(Task(watchService.take)(watchPool)).flatMap(s =>
-      Process.emitAll(s.pollEvents.asScala)).repeat
-  private val watchTopic: Topic[WatchEvent[_]] =
```

Comment:
The only place this was being used was down below where it immediately used `.subscribe` to get the `Process` again - I am assuming in scalaz-stream that `subscribe . topic = id`

Reply:
That seems legit.  I can imagine subtle differences in thread model.  Do we want our subscribers to operate on the `watchPool`?  Are they?  I'm asking more than reasoning.

Reply:
No clue, would that make a difference? Both `watchProcess` and `watchTopic` were on the `watchPool`

Reply:
Yes, I think it's going to be there anyway.  Let's try it your way.